samza-test/src/main/python/configs/downloads.json

```diff
@@ -1,5 +1,5 @@
 {
-  "url_kafka": "http://www.us.apache.org/dist/kafka/0.8.2.0/kafka_2.9.2-0.8.2.0.tgz",
-  "url_zookeeper": "http://archive.apache.org/dist/zookeeper/zookeeper-3.4.3/zookeeper-3.4.3.tar.gz",
+  "url_kafka": "http://archive.apache.org/dist/kafka/0.10.0.1/kafka_2.10-0.10.0.1.tgz",
```

Comment:
Json inherently doesn't support substitution of environment variable implicitly in values.  To accomplish this we should have a script (or a gradle task), which should generate the portion of json configuration that contains hardcoded scalaVersion based upon a ENV variable. 


Created a jira ticket to record this : https://issues.apache.org/jira/browse/SAMZA-1072